sdks/python/setup.py

```diff
@@ -300,25 +296,25 @@ def get_portability_package_data():
             # https://github.com/googleapis/google-cloud-python/issues/10566
             'google-auth>=1.18.0,<3',
             'google-auth-httplib2>=0.1.0,<0.2.0',
-            'google-cloud-datastore>=1.8.0,<2',
+            'google-cloud-datastore>=1.8.0,<3',
             'google-cloud-pubsub>=2.1.0,<3',
             'google-cloud-pubsublite>=1.2.0,<2',
             # GCP packages required by tests
             'google-cloud-bigquery>=1.6.0,<4',
-            'google-cloud-bigquery-storage>=2.6.3,<2.17',
+            'google-cloud-bigquery-storage>=2.6.3,<3',
             'google-cloud-core>=0.28.1,<3',
-            'google-cloud-bigtable>=0.31.1,<2',
+            'google-cloud-bigtable>=0.31.1,<3',
             'google-cloud-spanner>=3.0.0,<4',
             # GCP Packages required by ML functionality
             'google-cloud-dlp>=3.0.0,<4',
-            'google-cloud-language>=1.3.0,<2',
-            'google-cloud-videointelligence>=1.8.0,<2',
+            'google-cloud-language>=2.0,<3',
+            'google-cloud-videointelligence>=2.0,<3',
             'google-cloud-vision>=2,<4',
-            'google-cloud-recommendations-ai>=0.1.0,<0.8.0'
+            'google-cloud-recommendations-ai>=0.1.0,<1'
```

Comment:
let's keep the upper bound restricted by a minor version, until they release 1.x.

Reply:
Sounds good. I will make the change